### DISCUSSION TITLE
Different offsets in the game and in the editor

### DISCUSSION THREAD

**Discussion by poiqwe302** [ppy/osu#27507] [Ideas]
I'm playing with a big offset with no hitsounds (keyboard latency), and when I go into the editor I have to reset it because I'm building with hitsounds. Maybe a function can be added that will remove the offset in the editor or smth like this?

**Comment by Rekunan** [comment]
There are plans to have "setting profiles" in the future, so in this case you'd have 2 separate profiles for playing and editing, each with different offsets and whatever else u wanna have different.
